modules/popup-stack/README.md

```diff
@@ -0,0 +1,38 @@
+# Canvas Kit Popup Stack
+
+Stack for managing popup UIs to coordinate global concerns like escape key handling and rendering
+order. The popup stack is framework agnostic. The popup stack uses a shared array as the stack. This
+sharing is enforced. If the stack variable has already been declared, the first-instantiated will be
+used. There is no need to worry about bootstrapping at a specific time.
+
+All Popup-style UIs within a page should register with this stack in order to properly function
+together.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-popup-stack
```

Comment:
Any thoughts on whether this should be bundled in the bundle modules?

Reply:
Do you mean re-exported from `@workday/canvas-kit-react`?

Reply:
I don't think we'll re-export.